dspace-api/src/main/java/org/dspace/content/dao/impl/BitstreamDAOImpl.java

```diff
@@ -77,8 +77,7 @@
     public Iterator<Bitstream> findByCommunity(Context context, Community community) throws SQLException {
         Query query = createQuery(context, "select b from Bitstream b " +
                 "join b.bundles bitBundles " +
-                "join bitBundles.bundle bundle " +
-                "join bundle.items item " +
+                "join bitBundles.items item " +
```

Comment:
This change (and the others in this file) look unrelated to the feature. Please remove from the PR or explain their relevance.